include/tvm/tir/function.h

```diff
@@ -325,6 +325,13 @@ constexpr const char* kIsEntryFunc = "tir.is_entry_func";
  */
 constexpr const char* kIsGlobalFunc = "tir.is_global_func";
 
+/*!
+ * \brief Mark the function as run on the host, mutually exclusive with kTarget.
```

Comment:
When a function have kIsHostFunc attribute, it should not contain target attribute